ERROR: Option 'MS_LOG_TRACE' must have a value separated by equals sign. while meson setup

### Your environment

- Operating system: Windows Subsystem for Linux , ubuntu 1804
- Node version: v16.15.1
- npm version:  8.11.0
- gcc/clang version: gcc 7.5.0
- mediasoup version:  v3@94908

### Issue description
After executing `MESON_ARGS="-DMS_LOG_TRACE -DMS_LOG_FILE_LINE" npm i` with Debug buildtype,
it failed at `meson setup` with info as follow
```
> mediasoup@3.9.17 postinstall
> node npm-scripts.js postinstall

npm-scripts.js [INFO] running task "postinstall"
npm-scripts.js [INFO] executing command: node npm-scripts.js worker:build
npm-scripts.js [INFO] running task "worker:build"
npm-scripts.js [INFO] executing command: make -C worker
make: Entering directory '/mnt/d/docs/github_repo/mediasoup/worker'
/mnt/d/docs/github_repo/mediasoup/worker/out/pip/bin/meson setup \
        --prefix /mnt/d/docs/github_repo/mediasoup/worker/out/Debug \
        --bindir '' \
        --libdir '' \
        --buildtype debug \
        -Db_pie=true \
        -Db_staticpic=true \
        --reconfigure \
        -DMS_LOG_TRACE -DMS_LOG_FILE_LINE \
        /mnt/d/docs/github_repo/mediasoup/worker/out/Debug/build || \
        /mnt/d/docs/github_repo/mediasoup/worker/out/pip/bin/meson setup \
                --prefix /mnt/d/docs/github_repo/mediasoup/worker/out/Debug \
                --bindir '' \
                --libdir '' \
                --buildtype debug \
                -Db_pie=true \
                -Db_staticpic=true \
                -DMS_LOG_TRACE -DMS_LOG_FILE_LINE \
                /mnt/d/docs/github_repo/mediasoup/worker/out/Debug/build
ERROR: Option 'MS_LOG_TRACE' must have a value separated by equals sign.
ERROR: Option 'MS_LOG_TRACE' must have a value separated by equals sign.
Makefile:109: recipe for target 'setup' failed
make: *** [setup] Error 1
```

According to the err info, I try again with `MESON_ARGS="-DMS_LOG_TRACE=1" npm i`,
and got this
```
meson.build:1:0: ERROR: Unknown options: "MS_LOG_TRACE"

A full log can be found at /mnt/d/docs/github_repo/mediasoup/worker/out/Debug/build/meson-logs/meson-log.txt
Makefile:109: recipe for target 'setup' failed
make: *** [setup] Error 1
make: Leaving directory '/mnt/d/docs/github_repo/mediasoup/worker'
```

Here is /mnt/d/docs/github_repo/mediasoup/worker/out/Debug/build/meson-logs/meson-log.txt
```
Build started at 2022-06-23T14:01:28.392672
Main binary: /home/hu/anaconda3/bin/python3
Build Options: -Db_pie=true -Db_staticpic=true -DMS_LOG_TRACE=1 -Dprefix=/mnt/d/docs/github_repo/mediasoup/worker/out/Debug -Dbindir= -Dlibdir= -Dbuildtype=debug
Python system: Linux
The Meson build system
Version: 0.62.2
Source dir: /mnt/d/docs/github_repo/mediasoup/worker
Build dir: /mnt/d/docs/github_repo/mediasoup/worker/out/Debug/build
Build type: native build

meson.build:1:0: ERROR: Unknown options: "MS_LOG_TRACE"
```

So what is the problem, how can I fix it :)

--- a/worker/Makefile
+++ b/worker/Makefile
@@ -21,7 +21,7 @@ INSTALL_DIR ?= $(MEDIASOUP_OUT_DIR)/$(MEDIASOUP_BUILDTYPE)
 BUILD_DIR ?= $(MEDIASOUP_OUT_DIR)/$(MEDIASOUP_BUILDTYPE)/build
 MESON ?= $(PIP_DIR)/bin/meson
 # `MESON_ARGS` can be used to provide extra configuration parameters to Meson, such as adding defines or changing
-# optimization options. For instance, use `MESON_ARGS="-DMS_LOG_TRACE -DMS_LOG_FILE_LINE" npm i` to compile worker with
+# optimization options. For instance, use `MESON_ARGS="-Dms_log_trace=true -Dms_log_file_line=true" npm i` to compile worker with
 # tracing and enabled.
 #
 # NOTE: On Windows make sure to add `--vsenv` or have MSVS environment already active if you override this parameter.

--- a/worker/meson.build
+++ b/worker/meson.build
@@ -26,6 +26,18 @@ if host_machine.system() == 'windows'
   ]
 endif
 
+if get_option('ms_log_trace')
+  cpp_args += [
+    '-DMS_LOG_TRACE',
+  ]
+endif
+
+if get_option('ms_log_file_line')
+  cpp_args += [
+    '-DMS_LOG_FILE_LINE',
+  ]
+endif
+
 common_sources = [
   'src/lib.cpp',
   'src/DepLibSRTP.cpp',

--- a/worker/meson_options.txt
+++ b/worker/meson_options.txt
@@ -1,0 +1,2 @@
+option('ms_log_trace', type : 'boolean', value : false, description : 'When enabled, logs the current method/function if current log level is "debug"')
+option('ms_log_file_line', type : 'boolean', value : false, description : 'When enabled, all the logging macros print more verbose information, including current file and line')

